.github/ISSUE_TEMPLATE/release.md

```diff
@@ -23,20 +23,28 @@ examples of each step, assuming release vX.Y.0 is being cut.
 
 - [ ] **[In Crossplane Runtime]**: Prepared the release branch `release-X.Y` at the beginning of [Code Freeze]:
   - [ ] Created the release branch using the [GitHub UI][create-branch].
-  - [ ] Created and merged an empty commit to the `master` branch, if required to have it at least one commit ahead of the release branch.
-  - [ ] Run the [Tag workflow][tag-workflow] on the `master` branch with the release candidate tag for the next release `vX.Y+1.0-rc.0`.
+  - [ ] (On the **Master** Branch) Created and merged an empty commit, if required to have it at least one commit ahead of the release branch.
+  - [ ] (On the **Master** Branch) Run the [Tag workflow][tag-workflow-runtime] with the release candidate tag for the next release `vX.Y+1.0-rc.0`.
+  - [ ] (On the **Release** Branch) Run the [Tag workflow][tag-workflow-runtime] with the release candidate tag for the next release `vX.Y.0-rc.1` (assuming the latest rc tag for `vX.Y.0` is `vX.Y.0-rc.0`).
 - [ ] **[In Core Crossplane]:** Prepared the release branch `release-X.Y` at the beginning of [Code Freeze]:
   - [ ] Created the release branch using the [GitHub UI][create-branch].
-  - [ ] Created and merged an empty commit to the `master` branch, if required to have it at least one commit ahead of the release branch.
-  - [ ] Run the [Tag workflow][tag-workflow] on the `master` branch with the release candidate tag for the next release `vX.Y+1.0-rc.0`.
+  - [ ] (On the **Master** Branch) created and merged a PR bumping the Crossplane Runtime dependency to the release candidate tag from master, `vX.Y+1.0-rc.0`.
+  - [ ] (On the **Master** Branch) Run the [Tag workflow][tag-workflow] with the release candidate tag for the next release, `vX.Y+1.0-rc.0`.
+  - [ ] (On the **Release** Branch) created and merged a PR bumping the Crossplane Runtime dependency to the release candidate tag on the release branch, `vX.Y.0-rc.1`.
+- [ ] **[In Core Crossplane]:** Cut a Crossplane **release candidate** from the release branch `release-X.Y` at the beginning of [Code Freeze]:
+  - [ ] (On the **Release** Branch) Run the [Tag workflow][tag-workflow] with the release candidate tag for the release `vX.Y.0-rc.1` (assuming the latest rc tag for `vX.Y.0` is `vX.Y.0-rc.0`).
+  - [ ] (On the **Release** Branch) Run the [CI workflow][ci-workflow] and verified that the tagged build version exists on the [releases.crossplane.io] `build` channel, e.g. `build/release-X.Y/vX.Y.0-rc.1/...` should contain all the relevant binaries.
+  - [ ] (On the **Release** Branch) Run the [Promote workflow][promote-workflow] with channel `stable` and verified that the tagged build version exists on the [releases.crossplane.io] `stable` channel at `stable/vX.Y.0-rc.1/...`.
```

Comment:
worth noting here that the `--devel` helm flag is needed to actually install/verify the RC build on the `stable` channel?

maybe this can be done by linking to the install docs? e.g., https://docs.crossplane.io/v1.13/software/install/#install-pre-release-crossplane-versions

Reply:
I think that publishing to `stable` will make installations of the Crossplane CLI immediately start using the RC builds, e.g. the `install.sh` script uses the `stable` channel by default: https://github.com/crossplane/crossplane/blob/master/install.sh#L5

Not sure I'm particularly worried about it, and maybe it speaks to moving the CLI to its own repo soon anyways (tracking issue #4850), but thought I'd at least bring it up 🤔

Reply:
Good point, I was thinking about this as well. 

Ideally, we could extend the `install.sh` with a similar logic as helm, i.e. select a [pre-release](https://semver.org/#spec-item-9) version only if `--devel` is passed. Not sure how feasible that would be with a bash script, though.

However, I feel good about not being blocked by this now for the following reasons:
- This is the first time we're shipping most of the functionality there, and we will hardly break some existing workflow if someone installs the `v1.14.0-rc.1` unintentionally.
- This will be the case for a week until we push the actual release. 
- It is a cli, so, the impact would be limited.

Reply:
Looks like the install script just uses the `current`, which is managed by the [build submodule](https://github.com/upbound/build/blob/2672eeb767636ec837aa7c63cd7e26e6089fa810/makelib/output.mk#L64), i.e. latest promote'd become current. So, probably, an improvement there makes more sense.

Reply:
This should do the trick: https://github.com/crossplane/crossplane/pull/4903

Promote will put the artifacts into the stable channel but not set it as "current" if marked as a "pre-release" in the promote action UI.